.github/workflows/tests.yml

```diff
@@ -0,0 +1,40 @@
+name: Tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  workflow_dispatch:
+
+
+jobs:
+  test:
+    name: "Go ${{ matrix.go-version }} on ${{ matrix.os }}"
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
```

Comment:
I would not only test on Linux (`ubuntu-latest`), but also on macOS (`macos-latest`) and Windows (`windows-latest`).

Reply:
Thank you. I adjusted the setup according to your suggestions with 191fbc92796.